code/modules/clothing/under/shorts.dm

```diff
@@ -1,279 +1,144 @@
+	item_state = "shorts_jeans_mustang"
 
-/obj/item/clothing/under/shorts/jeans/mustang/female
+/obj/item/clothing/under/shorts/jeans/mustang/short
 	name = "mustang jeans short shorts"
-	icon_state = "jeansmustang_shorts_f"
-	item_state = "jeansmustang_shorts_f"
+	icon_state = "shorts_s_jeans_mustang"
+	item_state = "shorts_s_jeans_mustang"
 
 /obj/item/clothing/under/shorts/jeans/youngfolks
 	name = "young folks jeans shorts"
-	icon_state = "jeansyoungfolks_shorts"
-	item_state = "jeansyoungfolks_shorts"
+	icon_state = "shorts_jeans_youngfolk"
+	item_state = "shorts_jeans_youngfolk"
 
-/obj/item/clothing/under/shorts/jeans/youngfolks/female
+/obj/item/clothing/under/shorts/jeans/youngfolks/short
 	name = "young folks jeans short shorts"
-	icon_state = "jeansyoungfolks_shorts_f"
-	item_state = "jeansyoungfolks_shorts_f"
+	icon_state = "shorts_s_jeans_youngfolk"
+	item_state = "shorts_s_jeans_youngfolk"
 
 /obj/item/clothing/under/shorts/jeans/black
 	name = "black jeans shorts"
-	icon_state = "blackpants_shorts"
-	item_state = "blackpants_shorts"
+	icon_state = "shorts_jeans_black"
+	item_state = "shorts_jeans_black"
 
-/obj/item/clothing/under/shorts/jeans/black/female
+/obj/item/clothing/under/shorts/jeans/black/short
 	name = "black jeans short shorts"
-	icon_state = "black_shorts_f"
-	item_state = "black_shorts_f"
+	icon_state = "shorts_s_jeans_black"
+	item_state = "shorts_s_jeans_black"
 
 /obj/item/clothing/under/shorts/jeans/grey
 	name = "grey jeans shorts"
-	icon_state = "greypants_shorts"
-	item_state = "greypants_shorts"
+	icon_state = "shorts_jeans_grey"
+	item_state = "shorts_jeans_grey"
 
-/obj/item/clothing/under/shorts/jeans/grey/female
+/obj/item/clothing/under/shorts/jeans/grey/short
 	name = "grey jeans short shorts"
-	icon_state = "grey_shorts_f"
-	item_state = "grey_shorts_f"
-
-/obj/item/clothing/under/shorts/khaki
-	name = "khaki shorts"
-	desc = "For that island getaway. It's five o'clock somewhere, right?"
-	icon_state = "tanpants_shorts"
-	item_state = "tanpants_shorts"
+	icon_state = "shorts_s_jeans_grey"
+	item_state = "shorts_s_jeans_grey"
+/********** Jeans Shorts End **********/
 
-/obj/item/clothing/under/shorts/khaki/female
-	name = "khaki short shorts"
-	icon_state = "khaki_shorts_f"
-	item_state = "khaki_shorts_f"
-
-//pants
-
-/obj/item/clothing/under/pants
-	name = "jeans"
-	desc = "A nondescript pair of tough blue jeans."
-	icon = 'icons/obj/item/clothing/under/pants.dmi'
-	icon_state = "jeans"
-	item_state = "jeans"
-	contained_sprite = TRUE
-	gender = PLURAL
-	body_parts_covered = LOWER_TORSO|LEGS
-
-/obj/item/clothing/under/pants/track
-	name = "track pants"
-	desc = "Track pants, perfect for squatting."
-	icon = 'icons/obj/tracksuit.dmi'
-	icon_state = "trackpants"
-	item_state = "trackpants"
-	contained_sprite = TRUE
+/********** Athletic Shorts Start **********/
+// Athletic Shorts
+/obj/item/clothing/under/shorts/athletic
+	name = "athletic shorts parent item"
+	desc = DESC_PARENT
```

Comment:
Repeat desc. Also, all regular athletic shorts have unset descriptions. This will make ALL of them have desc as DESC_PARENT.